ext/drsyms/libelftc/rebuild_elftc.sh

```diff
@@ -47,11 +48,15 @@ fi
 # Fedora uses bmake.
 # We assume user has already patched according to
 # https://sourceforge.net/p/elftoolchain/wiki/BuildingFromSource/
-if [[ -e /etc/fedora-release ]]
+if [[ -e /etc/fedora-release  || $(uname -s) =~ MINGW.* ]]
 then
     make_cmd=bmake
+    echo "make cmd is bmake."
+    echo;
+    make_option="-B"
```

Comment:
It sounds like this makes building slower: could we add it only for mingw in that case.